### PR TITLE
libebml 1.3.1 / libmatroska 1.4.2

### PR DESCRIPTION
**Pull request by Noctem**
* libebml and libmatroska
  * Update to latest stable releases
  * Use Autotools on HEAD builds
  * Use new configure scripts
  * Modernize Formulas
* libmatroska
  * Apply upstream patch (Matroska-Org/libmatroska@9466bf5f2b) to link against libebml on stable builds
  * Use HTTPS on mirror link

### CODE REVIEW

File: Library/Formula/libmatroska.rb [hunk 1]
```diff
@@ -19,13 +22,61 @@ class Libmatroska < Formula
...
   else
-    depends_on 'libebml'
+    depends_on "libebml"
   end
 
+  depends_on "pkg-config" => :build
+
+  # Apply upstream patch to link against libEBML
+  # https://github.com/Matroska-Org/libmatroska/commit/9466bf5f2b
+  patch :DATA if build.stable?
```
Comment (jacknagel) [inline Library/Formula/libmatroska.rb]: Use a stable block rather than a conditional, please.

**Comment by Noctem** [comment]
Could I get a re-test on this and #36623? The failures don't appear to be related to the formulas.

**Comment by mikemcquaid** [comment]
@Noctem Done. Shout when they're done and/or green.

**Comment by Noctem** [comment]
@mikemcquaid The bot appears to be confused. It's trying to build *mkvtoolnix*, and then it's failing to download a non-existent *libmatroska 1.4.2* bottle.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**